src/components/Settings/Setting.tsx

```diff
@@ -23,6 +27,39 @@ const NavLink = ({ to, title }) => {
   );
 };
 
+const TenantLicenses = () => {
+  const { data: licenses, isLoading, error } = useFetchLicenses();
+  return (
+    <Stack
+      direction="row"
+      spacing={4}
+      sx={{
+        justifyContent: "flex-end",
+        alignItems: "center",
+        pt: "1rem",
+      }}
+    >
+      {isLoading ? (
+        <CircularProgress size="sm" />
+      ) : (
+        <>
+          {licenses?.licensedRepos != undefined && licenses?.licensedRepos != null && (
```

Comment:
This can be shortened as follows, right?
```suggestion
          {licenses?.licensedRepos && (
```

Reply:
Ok, thanks for explanation.